### PR TITLE
Remove guice dependency

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

File: shaded/hadoop/pom.xml [hunk 1]
```diff
@@ -142,9 +142,6 @@
                 <relocation>
                   <pattern>com.google</pattern>
                   <shadedPattern>${shading.prefix}.com.google</shadedPattern>
-                  <excludes>
```
Comment (gpang) [inline shaded/hadoop/pom.xml:145]: Does this mean that all of these packages will be shaded now? Why were they not shaded before?

Comment (yuzhu) [reply shaded/hadoop/pom.xml:145]: actually guice has not been included in hdfs package for a long time, so it did not have an effect. 
not sure why it was shaded before, @LuQQiu do you know?

Comment (yuzhu) [reply shaded/hadoop/pom.xml:145]: discussed offline, not sure why it was excluded before. but i think we should shade it.

**Comment by yuzhu** [comment]
@gpang PTAL, updated comments on shading exclusion

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**